configuration-sample/ods-core.env.sample

```diff
@@ -189,6 +189,9 @@ JENKINS_AGENT_BASE_SNYK_DISTRIBUTION_URL=https://github.com/snyk/snyk/releases/d
 # You may also point to repository underneath REPO_BASE.
 SHARED_LIBRARY_REPOSITORY=https://github.com/opendevstack/ods-jenkins-shared-library.git
 
+# Comma-separated list of group type and group name to be added as role to a project on project provision (see create-project.sh)
+#ODS_CLUSTER_GROUPS=ADMINGROUP=<YOUR_CLUSTER_ADMIN_GROUP>
```

Comment:
Shouldn't be commented - let's have the example in a comment, and the default be the empty value.